proto/internal/temporal/server/api/persistence/v1/workflow_mutable_state.proto

```diff
@@ -72,6 +72,7 @@ message WorkflowMutableStateMutation{
     repeated int64 deleted_signal_infos = 12;
     repeated string deleted_update_infos = 13;
     repeated StateMachinePath deleted_sub_state_machines = 14;
+    repeated StateMachineTombstoneBatch sub_state_machine_tombstone_batches = 95;
```

Comment:
we can exclude this field from the `execution_info` field and let's also update the comment on L78 for that. 

plz also update the field number.